src/blocks/cards.scss

```diff
@@ -0,0 +1,103 @@
+html {
+  color: var(--color-title-rev-price);
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin-bottom: 46px;
+  margin-right: 48px;
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__image {
+    margin-bottom: 40px;
+    align-self: center;
+    vertical-align: top;
+    width: 160px;
+    height: 134px;
+    padding: 0;
+  }
+
+  &__title {
+    font-weight: 500;
+    margin-bottom: 4px;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    line-height: 14px;
+    color: var(--grey-color);
+    margin-bottom: 16px;
+    font-style: normal;
+  }
+
+  &__price-and-value {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    align-items: center;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 23px;
+    align-items: center;
+    height: 16px;
+  }
+
+  &__rev {
+    font-weight: 400px;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    font-size: 12px;
+    font-weight: 400px;
+    color: var(--grey-color);
+  }
+
+  &__value {
+    font-weight: 700;
+    font-size: 16px;
+  }
+
+  &__button {
+    text-align: center;
+    background-color: var(--button-color);
+    border-radius: 5px;
+    padding-top: 11px;
+    padding-bottom: 11px;
+    color: var(--button-second-color);
+    border: 1px solid transparent;
+
+    transition: 0.2s;
```

Comment:
add properties for which you want to use transition (by default it'll be applied to all properties, which will affect in bad way on website performance)